packages/ia-wayback-search/README.md

```diff
@@ -44,8 +44,9 @@ https://modern-web.dev/docs/test-runner/overview/
 
 ## Structure
 
+
 * index.js - main component export
-* index.html - file opened when running `yarn start`
+* `yarn start` - runs local server in Demo directory is in `/www` folder
 * /src/wayback-search.js - main component definition
```

Comment:
personally i wouldnt mind both since i sometimes dont wanna deal w/ yarn/npm for just opening a HTML file w/ JS imports.  YMMV...

Reply:
ok, can update

Reply:
ok, also referenced npm 👍